front/src/modules/project/components/AddOrEditProjectModal.tsx

```diff
@@ -118,8 +118,23 @@ export default function AddOrEditProjectModal({
     }
   };
 
+  const currentProjectNameIsInvalid: boolean =
+    !currentProject.name || currentProject.name.length > 128;
+  const currentProjectObjectivesIsInvalid: boolean =
+    (currentProject.objectives ?? '').length > 4096;
+  const currentProjectDescriptionIsInvalid: boolean =
+    (currentProject.description ?? '').length > 1024;
+  const currentProjectFundersIsInvalid: boolean = (currentProject.funders ?? '').length > 255;
+  const currentProjectBudgetIsInvalid: boolean = (currentProject.budget ?? 0) > 2147483647;
+
   const createProject = async () => {
-    if (currentProject.name === '') {
+    if (
+      currentProjectNameIsInvalid ||
+      currentProjectObjectivesIsInvalid ||
+      currentProjectDescriptionIsInvalid ||
+      currentProjectFundersIsInvalid ||
+      currentProjectBudgetIsInvalid
```

Comment:
You can use directly your const hasError here
```suggestion
     hasError
```

Reply:
Done